### ISSUE TITLE
[Common Lisp] Implement new Concept Exercise: multiple-values

### ISSUE THREAD

**Issue by TheLostLambda** [exercism/v3#1827]
This issue describes how to implement the `multiple-values` concept exercise for the Common Lisp track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Learning objectives

- Understand that functions in Common Lisp can return multiple values with `values`
- Know that the first return value is the "default" and others need to be extracted with `multiple-value-*`
- Be aware of some standard functions that return multiple values and what those values mean (`floor`, `gethash`, etc.)
- Be able to convert multiple values into a list using `multiple-value-list`
- Know how to destructure multiple values using `multiple-value-bind`
- Be able to extract a particular return value by index using `nth-value`

## Out of scope

- The more "obscure" multiple value functions (`multiple-value-prog1`, `multiple-value-call`, `values-list`, etc)
- Detailed explanations of multiple value returning functions like `gethash` or their context (hash-tables)
- Destructuring of single values (using `destructuring-bind`)

## Concepts

- `multple-values`

## Prerequisites

- `basic-lists`
- `arithmetic`
- `variables`
- `function-definition`
- `expressions`

## Resources to refer to

- [List of Functions](https://www.cs.cmu.edu/Groups/AI/html/cltl/clm/node94.html)

### Hints

- [CL Cookbook](https://lispcookbook.github.io/cl-cookbook/functions.html#multiple-return-values-values-multiple-value-bind-and-nth-value)

### After

- [Rational Behind Multiple Values (vs a list)](https://stackoverflow.com/questions/22795608/values-function-in-common-lisp/22796346#22796346)

## Representer

Nothing yet

## Analyzer

Multiple values are pretty easy to pick up in testing, so not much here. Maybe check both values from `floor` are being used vs just the first value of `floor` and something like `mod` or `rem`.

## Implementing

Be sure the emphasize that returning multiple values is quite different from returning a list. You could show how a function can be changed to return multiple values but still be used as if it only returned one without any additional code changes.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.
